apps/web/src/AppRoutes.tsx

```diff
@@ -91,6 +93,8 @@ export const AppRoutes = () => {
       <Route element={<PrivatePageLayout />}>
         <Route path={ROUTES.WORKFLOWS_DIGEST_PLAYGROUND} element={<TemplatesDigestPlaygroundPage />} />
         <Route path={ROUTES.WORKFLOWS_CREATE} element={<TemplateEditorPage />} />
+        <Route path={ROUTES.WORKFLOWS_V2_STEP_EDIT} element={<WorkflowsStepEditorPageV2 />} />
```

Comment:
I've introduced a new page for the step editor, as I think that the new name makes sense more than the old one and simplifies execution quite a lot.